t/contexts/significant_figures.t

```diff
@@ -23,254 +23,364 @@ use Data::Dumper;
 
 Context('SignificantFigures');
 
+sub ROUND  {&context::SignificantFigures::Real::ROUND}
+sub expon  {&context::SignificantFigures::Real::expon}
+sub FORMAT {&context::SignificantFigures::Real::format}
```

Comment:
With my recommended changes, this would be

``` perl
sub expon { context::SignificantFigures::Real->expFor($_[0], 0) }
sub FORMAT { context::SignificantFigures::Real->format(@_) }